it_IT.md

```diff
@@ -54,37 +54,36 @@ Il Design è una soluzione intenzionale ad un problema ottenuta operando all'int
 
 E l'empatia? Empatia è una bella parola da usare al posto di "esclusione". Se vuoi capire come una donna userebbe qualcosa che stai progettando, aggiungi una donna al gruppo di lavoro che sta lavorando al progetto.
 
-### A designer does not believe in edge cases.
+### Un designer non crede nei casi limite.
 
-When you decide who you’re designing for, you’re making an implicit statement about who you’re not designing for. For years we referred to people who weren’t crucial to our products’ success as “edge cases”. We were marginalizing people. And we were making a decision that there were people in the world whose problems weren’t worth solving.
+Quando decidi per chi stai progettando, stai facendo una dichiarazione implicita su chi non stai progettando. Per anni abbiamo chiamato “casi limite” le persone che non erano determinanti per il successo dei nostri prodotti. Stavamo emarginando le persone. E stavamo prendendo la decisione che c'erano persone nel mondo i cui problemi non valevano la pena di essere risolti.
 
-Facebook now claims to have two billion users. 1% of two billion people, which most products would consider an edge case, is twenty million people. Those are the people at the margins.
+Facebook ora afferma di avere due miliardi di utenti. L'1% di due miliardi di persone, che la maggior parte dei prodotti considererebbe un caso limite, è di venti milioni di persone. Quelle sono le persone ai margini.
 
-“When you call something an edge case, you’re really just defining the limits of what you care about.” — Eric Meyer
-These are the trans people who get caught on the edges of “real names” projects. These are the single moms who get caught on the edges of “both parents must sign” permission slips. These are the elderly immigrants who show up to vote and can’t get ballots in their native tongues.
+"Quando chiami qualcosa un caso limite, stai solo definendo i limiti di ciò a cui tieni". — Eric Meyer. Queste sono le persone trans che rimangono intrappolate ai margini dei progetti con "nomi veri". Queste sono le mamme single che rimangono intrappolate ai margini delle schede di autorizzazione "entrambi i genitori devono firmare". Questi sono gli immigrati anziani che si presentano per votare e non possono ottenere schede nelle loro lingue native.
```

Comment:
Could you put "Queste sono le persone..." on a separate bullet note to mirror the original?

Reply:
done! thanks for the catch!